src/OrchardCore/OrchardCore.DisplayManagement/Entities/SectionDisplayDriver.cs

```diff
@@ -18,14 +19,17 @@ public abstract class SectionDisplayDriver<TModel, TSection> : DisplayDriver<TMo
         where TSection : new()
         where TModel : class, IEntity
     {
+        [Obsolete("Instead use SectionName")]
+        protected virtual string PropertyName => typeof(TSection).Name;
```

Comment:
I changed it back to `PropertyName` to avoid having to debate which is a better name. But I think SectionName would make more sense here since we are talking about section not property.